ansible/roles/tomcat/tasks/restart.yml

```diff
@@ -44,6 +44,7 @@
   environment:
     CATALINA_BASE: "{{ catalina_base }}"
     CATALINA_PID: "{{ catalina_base }}/tomcat.pid"
+    UMASK: "{{ tomcat_umask }}"
```

Comment:
If we go that route, `run` below should probably be updated too.

But shouldn't this be done in the `setenv.sh.j2` template instead, so that starting Tomcat outside of Ansible does the right thing too (I'm wondering if we could also put `CATALINA_BASE` and `CATALINA_PID` in there to simplify the manual run command)?

Reply:
setting UMASK in setenv.sh.j2 is working perfectly.
I will try to set CATALINA_BASE and CATALINA_PID as well later to have shorter manual cmds (and to please my boss)